sqlmesh/dbt/context.py

```diff
@@ -238,9 +238,13 @@ def context_for_dependencies(self, dependencies: Dependencies) -> DbtContext:
             else:
                 raise ConfigError(f"Source '{source}' was not found.")
 
+        variable_deps = dependencies.variables
```

Comment:
Are you using `variable_deps` so that the next piece fits on one line?

Reply:
mostly, yes

Reply:
My personal preference is not to introduce extra variables. black does a good enough job imo. Up to you though.